frontend/test/lit/test_peephole_optimizations.py

```diff
@@ -87,6 +87,42 @@ def test_pipeline_lowering_workflow(x):
 test_pipeline_lowering()
 
 
+def test_transform_lowering():
+    """
+    Basic pipeline lowering on one qnode.
+    """
+
+    @qjit(keep_intermediate=True)
+    @qml.transforms.merge_rotations
+    @qml.transforms.cancel_inverses
+    @qml.qnode(qml.device("lightning.qubit", wires=2))
+    def test_pipeline_lowering_workflow(x):
+        qml.RX(x, wires=[0])
+        qml.Hadamard(wires=[1])
+        qml.Hadamard(wires=[1])
+        return qml.expval(qml.PauliY(wires=0))
+
+    # CHECK: pipeline=(<cancel_inverses((), {})>, <merge_rotations((), {})>)
+    print_jaxpr(test_pipeline_lowering_workflow, 1.2)
+
+    # CHECK: transform.named_sequence @__transform_main
+    # CHECK-NEXT: {{%.+}} = transform.apply_registered_pass "remove-chained-self-inverse" to {{%.+}}
```

Comment:
```suggestion
    # CHECK-NEXT: {{%.+}} = transform.apply_registered_pass "cancel-inverses" to {{%.+}}
```
we recently renamed this